extenstions/avahi-debian/install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -e
+
+apt-get update && apt-get install -y --no-install-recommends avahi-daemon libnss-mdns
+
+# allow hostnames with more labels to be resolved. so that we can
+# resolve node1.mycluster.local.
+# (https://github.com/lathiat/nss-mdns#etcmdnsallow)
+echo '*' > /etc/mdns.allow
+
+# Configure NSSwitch to use the mdns4 plugin so mdns.allow is respected
+sed -i "s/hosts:.*/hosts:          files mdns4 dns/g" /etc/nsswitch.conf
+printf "[server]\nenable-dbus=no\n" >> /etc/avahi/avahi-daemon.conf
+chmod 777 /etc/avahi/avahi-daemon.conf
```

Comment:
Why are `777`s required here and below? Can we somehow get along using stricter file permissions?

Reply:
`755` should be enough. Fixed.